latest_requirements.txt

```diff
@@ -0,0 +1,16 @@
+XlsxWriter==3.2.0
+appdirs==1.4.4
+boto3==1.34.55
+botocore==1.34.55
+compress-pickle==2.1.0
+humanfriendly==10.0
```

Comment:
@gsheni Should we add all these dependencies or only the major ones?

Reply:
Let's add all of them for now. If it gets to be too much, we can remove some.